### PR TITLE
refactor(Palette) Updates to palette since doesn't follow a theme correctly

### LINKED ISSUE #121: Update to use current UX defined Palette
The palette used by Octagon differs from current TW UX guidelines, we should upgrade it.

Specifically, twgrey is no longer to be used, we've generally replaced this with #28324E except where we've used one of #A6A8AB or #28324E.

This will need some review by TOC as well as UX teams.

### PR DESCRIPTION
**Pull request by andrewleyva**
# problem statement
The current Octagon palette doesn't follow Yung or Antoine's themes on UX Pin. Octagon and TOC have begun to diverge. 

# solution
Assign variables to colors and to components and types to enable more readability and reuse. Update colors where needed.

# discussion
This is step one in a long process of bringing consistency across products.

Closes #121

### CODE REVIEW

**Comment by wa11-e** [comment]
Woohoo! new surge deployment available for viewing! :tada: [surgereview30006154.surge.sh](http://surgereview30006154.surge.sh)

**Comment by wa11-e** [comment]
Woohoo! new surge deployment available for viewing! :tada: [surgereview30008045.surge.sh](http://surgereview30008045.surge.sh)

File: src/components/native/LargeCard/LargeCardTitle.jsx [hunk 1]
```diff
@@ -8,7 +8,7 @@ const LargeCardTitle = (props) => {
   return (
     <Flexbox {...externalAttributes} flexGrow={3} flexDirection='column'>
       <div title={props.title} className='title'>{props.title}</div>
-      <div title={props.description} className='text-small textGray'>{props.description}</div>
+      <div title={props.description} className='friendly-name'>{props.description}</div>
```
Comment (ggascoigne) [inline src/components/native/LargeCard/LargeCardTitle.jsx]: class names use underscores.

File: src/styles/components/notification-item.css [hunk 2]
```diff
@@ -474,11 +473,11 @@ box-shadow: -3px 0px 9px -4px rgba(0,0,0,0.31);
 .all__notifications{
   margin-right: 7px;
   cursor: pointer;
-  color: var(--twgray);
+  color: var(--twGray);
```
Comment (ggascoigne) [inline src/styles/components/notification-item.css]: I think that twGray should go away.

File: src/components/native/LargeCard/LargeCardTitle.jsx (hunk 1 above)
Comment (andrewleyva) [reply src/components/native/LargeCard/LargeCardTitle.jsx]: I'll update classes to snake_case.

File: src/styles/components/notification-item.css (hunk 2 above)
Comment (andrewleyva) [reply src/styles/components/notification-item.css]: I left it alone in the components we aren't using yet. I can swap it with DimGray (#666666) for now.

**Comment by wa11-e** [comment]
Woohoo! new surge deployment available for viewing! :tada: [surgereview30146719.surge.sh](http://surgereview30146719.surge.sh)

**Comment by wa11-e** [comment]
Woohoo! new surge deployment available for viewing! :tada: [surgereview30146895.surge.sh](http://surgereview30146895.surge.sh)

**Pull request merged**